File: 13-swap-case/swap-case.js
const swapCase=(str)=>{

    const result=[]

    for(var i=0; i<str.length;i++){
        if(str[i]==str[i].toUpperCase()){
            result.push(str[i].toLowerCase())
        }else{
            result.push(str[i].toUpperCase())
        }
    }

    return result.join('')
}

console.log(swapCase('Luke, I Am Your Father'))

File: 13-swap-case/swap_case.py
def swap_case(string):

    result=[]

    for character in string:
        if character==character.lower():
            result.append(character.upper())
        else:
            result.append(character.lower())
    
    return ''.join(result)


print(swap_case('ImAgInE If i WrOtE LiKe tHiS'))